util/test_timeline.py

```diff
@@ -300,15 +317,23 @@ def check_event(event, timelist, state):
             if not state['timeline_stopped']:
                 for other_entry in timelist:
                     if (
-                            'regex' in other_entry and
-                            other_entry['time'] > state['last_jump'] and
-                            other_entry['time'] < entry['time'] and
-                            other_entry['branch'] < entry['branch']
+                        'regex' in other_entry
+                        and other_entry['time'] > state['last_jump']
```

Comment:
I think this "operator at the start of the line" thing is the most personally distasteful style thing I see in this whole change, but I will take it at the expense of automatic formatting for everything else.

Reply:
Yea, some of it is gross.. We can still change courses and divert to something else if you have another preference

Reply:
In other news, I'm changing all my if statements from
```python
if x
  and y
```
to
```python
if x:
  if y:
```

Reply:
Yeah, I haven't investigated a ton of other ones.  Did you have anything else in mind, or should I go looking?